ludwig/utils/tokenizers.py

```diff
@@ -50,14 +46,61 @@ def __call__(self, text: str):
         pass
 
 
-class CharactersToListTokenizer(BaseTokenizer):
-    def __call__(self, text):
-        return [char for char in text]
+class StringSplitTokenizer(torch.nn.Module):
```

Comment:
Let's add a test for this tokenizer in `tokenizer_test.py`.

In case you haven't done this on Ludwig yet -- tests can be run with `pytest <python_file>::name_of_test` e.g. 

```
pytest tests/ludwig/utils/test_tokenizers.py::test_english_lemmatize_filter_tokenizer
```

Reply:
Good call! Done!